x-pack/plugins/spaces/server/lib/request_interceptors/on_request_interceptor.test.ts

```diff
@@ -6,6 +6,8 @@
  */
 
 import { schema } from '@kbn/config-schema';
+import type { createRoot } from '@kbn/core-test-helpers-kbn-server';
+import { request } from '@kbn/core-test-helpers-kbn-server';
```

Comment:
Same thing here if you like:

```suggestion
import { type createRoot, request } from '@kbn/core-test-helpers-kbn-server';
```